data/js/values/values.js

```diff
@@ -77,4 +83,8 @@
         }
     });
 
+    nextButton.addEventListener('click', function() {
+        // notify addon that we've clicked the next button
+        addon.port.emit('next_selected');
+    });
```

Comment:
There is a lot of repetition of this code block as well as:

```
...
nextButton.classList.remove('hidden');
...
```
This can easily cause a maintenance problem where we forget to update one or more of the instances. Could we turn this into a utility function that each of these can then simply call?